src/app/routes/api/v3/forms/public-forms.verification.routes.ts

```diff
@@ -58,4 +60,7 @@ PublicFormsVerificationRouter.route(
  */
 PublicFormsVerificationRouter.route(
   '/:formId([a-fA-F0-9]{24})/fieldverifications/:transactionId([a-fA-F0-9]{24})/fields/:fieldId([a-fA-F0-9]{24})/otp/generate',
-).post(VerificationController.handleGenerateOtp)
+).post(
+  limitRate({ max: rateLimitConfig.sendAuthOtp }),
```

Comment:
I think we need to apply the same limit on the old endpoint `/:transactionId([a-fA-F0-9]{24})/otp`

Reply:
thanks! i added it